### PR TITLE
iOS amplify-app / init quickstart fix

### PR DESCRIPTION
**Pull request by edwardfoyle**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-app/src/call-amplify.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const callPkgAmplifyNix = async (args, opts) => {
 const callNodeAmplify = async (args, opts) => {
   opts = { ...defaultOpts, ...opts };
   const amplifyCmd = path.basename(process.argv[1]) === 'amplify-app-dev' ? amplifyDev : amplify;
-  const { stdout, stderr } = await execa(amplifyCmd, args, { stdio: opts.inheritIO ? 'inherit' : undefined });
+  const { stdout, stderr } = await execa('amplify-dev', args, { stdio: opts.inheritIO ? 'inherit' : undefined });
```
Comment (edwardfoyle) [inline packages/amplify-app/src/call-amplify.js:39]: need to revert this

File: packages/amplify-frontend-ios/package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "test-watch": "jest --watch"
   },
   "dependencies": {
+    "amplify-app": "2.18.1",
```
Comment (kaustavghosh06) [inline packages/amplify-frontend-ios/package.json:23]: Need to change this to the latest - 2.19.2

File: packages/amplify-app/src/project-configs/project-config-android.json [hunk 3]
```diff
@@ -1,11 +0,0 @@
-{
-  "projectName": "amplifyStarter",
-  "version": "2.0",
-  "frontend": "android",
-  "android": {
-    "config": {
-      "ResDir": "app/src/main/res"
```
Comment (kaustavghosh06) [inline packages/amplify-app/src/project-configs/project-config-android.json:7]: Where do we store this info now?

Comment (edwardfoyle) [reply packages/amplify-app/src/project-configs/project-config-android.json:7]: it's now in amplify-cli (the android stuff has always been there, but now ios and javascript is there as well)

**Review by kaustavghosh06 — COMMENTED** [review]
LGTM. Just take a look at the 2 comments - that need to be addressed before merging.

**Comment by edwardfoyle** [comment]
@kaustavghosh06 bumped the version and responded to the other comment

File: packages/amplify-app/src/index.js [hunk 4]
```diff
@@ -98,82 +137,68 @@ async function amplifyCLIVersionCheck() {
...
-  projectConfig = JSON.parse(fs.readFileSync(projectConfigTemplateFilePath));
-  projectConfig.projectName = projectName;
-
-  if (suitableFrontend === 'javascript') {
-    projectConfig.javascript.framework = jsFrameWork;
-    projectConfig.javascript.config = frameworkConfigMapping[jsFrameWork];
+  if (framework && projectConfig.javascript) {
+    projectConfig.javascript.framework = framework;
+    projectConfig.javascript.config = frameworkConfigMapping[framework];
+    fs.writeFileSync(projectConfigFilePath, JSON.stringify(projectConfig, null, 4));
```
[the remark this answers is not in the document]
Comment (edwardfoyle) [reply packages/amplify-app/src/index.js:172]: amplify-app doesn't (and can't) depend on amplify-cli or amplify-cli-core

File: packages/amplify-app/src/index.js [hunk 5]
```diff
@@ -98,82 +137,68 @@ async function amplifyCLIVersionCheck() {
...
+    console.log(`${emoji.get('x')} Failed to create base Amplify Project`);
+    throw new Error(err);
   }
+};
 
-  const projectName = path.basename(process.cwd());
-  let projectConfig = {};
+const updateFrameworkInProjectConfig = framework => {
+  const projectConfigFilePath = path.join('amplify', '.config', 'project-config.json');
+  const projectConfig = JSON.parse(fs.readFileSync(projectConfigFilePath, 'utf8'));
```
[the remark this answers is not in the document]
Comment (edwardfoyle) [reply packages/amplify-app/src/index.js:167]: same as above

**Pull request merged**